torat_server/shell.go

```diff
@@ -27,10 +24,10 @@ func Shell() {
 	cwd, err := os.Getwd()
 	if err != nil {
 		panic(err)
+	} else {
```

Comment:
This else is useless, panic exits before anyway.

Reply:
I see this as a difference between writing styles. By making the error a switch it makes the logic more contained and easier for me to read. The compiler also will compile this, in the same way, give the surrounding code. 

I can change this if you feel strongly about it, but you may see this style by me elsewhere by force of habit.

Reply:
Yes I feel strongly about that, because it reduces nesting.

Reply:
I'm not sure I understand your reasoning but you will prob see me doing this elsewhere. At this point, though I just want to get even 50% of the features working. We can clean up the little stuff as we go